pkgs/development/python-modules/jaxlib/bin.nix

```diff
@@ -27,6 +27,7 @@
 , scipy
 , stdenv
   # Options:
+, ml-dtypes
```

Comment:
please move this out of the "Options" section and into the arguments above in alphabetical order

Reply:
Done